jib-gradle-plugin/CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ All notable changes to this project will be documented in this file.
 ### Added
 
 ### Changed
+- `jib.container.creationTime` now accepts more timezone formats:`Z`,`+HHmm`. ([#2320](https://github.com/GoogleContainerTools/jib/issues/2260))
```

Comment:
I believe `Z` was already accepted. So, "accepts a timezone format without colon: `+HHmm`"? And if this improves usability with git, worth nothing that? Like, "This helps use git commit timestamp from `git ...` or some git timestamp plugin"?

Reply:
Oh, the issue number in `issues/2260` is wrong.

Reply:
nit: also add a space after `### Changed`. I saw we have the same problem in line 15, so can you add a space after `### Added` there too?